config.yaml

```
@@ -325,6 +328,46 @@ orgs:
         privacy: closed
         repos:
           ansible-snapshot: admin
+      ansible-zipship:
+        description: "Ansible ZipShip"
+        maintainers:
+          - A-Beck
+          - sabre1041
+          - bontreg
```

I you missed this one @sabre1041 
```suggestion
          - bontreger
```

---

3rd time is the charm